src/Routes/RosPage/RosPage.js

```diff
@@ -115,6 +116,11 @@ class RosPage extends React.Component {
                 return { label: os, value: os.split(' ')[1] };
             });
 
+            if (osObject.filterValues.items.length === 0) {
+                osObject.filterValues.items = [{ value: '', label: 'No versions available' }];
+                osObject.type = conditionalFilterType.group;
+            }
```

Comment:
@r14chandra I feel we should either use `string` here for type or make it same for `checkbox` type for consistency. what you think?

Reply:
Why do we need a checkbox for this scenario?

Reply:
no not for this scenario but in the same method we first declare the type using string value. So I was saying may be use either direct string or constant at both places. let me know if you have more questions @r14chandra 

`osObject.label = 'Operating system';
 osObject.type = 'checkbox';`

Reply:
Direct string won't work for `group` type.

Reply:
**Update:** Yes, that also works. I have updated the `checkbox` type to fetch its value from `conditionalFilterType` as well.